app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/models/EnumJson.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformmicroservice.apidefinition.models
```

Comment:
2.7 now has enumeration readers as part of standard play json IIRC

Reply:
Also part of Enumeratum - https://github.com/lloydmeta/enumeratum#play-json

Reply:
Do you use play-json-2.7 with the rest of the project being on play-2.6? I think you told me before it was fine. If you haven't seen any issues with that I don't mind using that. I've never used Enumeratums but if that's the way to go I'm fine with that as well. Don't really have a preference at this point.

Reply:
Yup. 2.7 play json works fine with play 2.6.